cmd/flags.go

```diff
@@ -35,6 +36,8 @@ func initFlags(
 		switch category {
 		case config.Ethereum:
 			initEthereumFlags(cmd, cfg)
+		case config.Electrs:
```

Comment:
If we do in the config `[bitcoin.electrum]`, then this one should probably become:

```suggestion
		case config.BictoinElectrum:
```

Reply:
Done in b26f5297258e43a7f57047a0157e5f3b111adab5.